src/models/db.ts

```diff
@@ -1,13 +1,17 @@
 import Dexie, { Table } from 'dexie';
 import { FeelingItem } from './FeelingItem';
+import { GoalItem } from './GoalItem';
 
 export class FeelingsDB extends Dexie {
   feelingsCollection!: Table<FeelingItem, number>;
 
```

Comment:
Remove this blank line here

Reply:
Actually, I can't because eslint is throwing an error **Expected blank line between class members**